boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -145,6 +145,32 @@ arduino_i2c: &i2c1 {
 	pinctrl-names = "default", "sleep";
 };
 
+&spi4 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	pinctrl-0 = <&spi4_default>;
+	pinctrl-1 = <&spi4_sleep>;
+	pinctrl-names = "default", "sleep";
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+	mx25r64: mx25r6435f@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <80000000>;
```

Comment:
I guess `spi-max-frequency = <DT_FREQ_M(8)>;` would apply here. BTW. Are we sure this is the highest possible frequency for SPI4?

Reply:
This flash chip can handle 80 MHz only in the High Performance Mode and currently the spi_nor driver is not capable of enabling this mode. In the default Ultra Low Power Mode, the maximum frequency is 8 MHz.